### PR TITLE
Kill Travis builds that are running for over 1 hour

### PR DESCRIPTION
**Pull request by bdunne**
Sometimes Travis builds get stuck and Travis doesn't kill them.  This can result in builds running for 18+ hours before someone notices and manually kills them.  Since that worker was tied up, the Travis backlog continues to grow.

### CODE REVIEW

**Comment by bdunne** [comment]
@jrafanie @Fryguy Please review.

cc @kbrock

**Comment by kbrock** [comment]
@bdunne cool.

Should we put something different around the vmdb build? that seems to be teetering around 1 hour mark. Maybe bump that up to 1 1/2 or 2?

**Comment by kbrock** [comment]
misspoke. vmdb takes 30 minutes.
Do we want to not kill master? or can we restart those?

**Comment by bdunne** [comment]
re-working... this should actually look at the job, not the build

File: app/workers/travis_build_killer.rb [hunk 1]
```diff
@@ -47,6 +45,12 @@ def kill_builds
           b.cancel
         end
       end
+
+      long_running_jobs.each do |b|
+        next unless b.running?
```
Comment (Fryguy) [inline app/workers/travis_build_killer.rb]: How is this line ever not running?

Comment (bdunne) [inline app/workers/travis_build_killer.rb]: If we cancel it above in the `out_dated_builds`

Comment (Fryguy) [inline app/workers/travis_build_killer.rb]: But the first thing you do on https://github.com/ManageIQ/miq_bot/pull/132/files#diff-d80c65f3a5fe96f44114c253eb74c918R35 is to only select the running ones.

Comment (Fryguy) [inline app/workers/travis_build_killer.rb]: Oh, if you cancel it, in between?...that's really confusing...perhaps instead, do the `long_running_jobs =` line after you do the outdated builds?

Comment (Fryguy) [inline app/workers/travis_build_killer.rb]: Or remove the outdated_builds from the long_running_jobs ?

**Comment by bdunne** [comment]
@Fryguy updated.

**Pull request merged**